packages/video_player/video_player_android/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayer.java

```diff
@@ -317,6 +351,74 @@ void sendInitialized() {
     }
   }
 
+
+  List<Messages.GetEmbeddedSubtitlesMessage> getEmbeddedSubtitles() {
+    List<Messages.GetEmbeddedSubtitlesMessage> subtitleItems = new ArrayList<>();
+    TrackGroupArray trackGroups;
+    int rendererIndex = 2;
+
+    MappingTrackSelector.MappedTrackInfo trackInfo = trackSelector.getCurrentMappedTrackInfo();
```

Comment:
I would suggest also ensuring here that `trackSelector` is not null like below in `setEmbeddedSubtitles()`.